swap_meet/vendor.py

```diff
@@ -1,2 +1,110 @@
+from .item import Item
+
 class Vendor:
-    pass
\ No newline at end of file
+    def __init__(self,inventory = None):
+        if inventory is None:
+            inventory = []
+        self.inventory = inventory
+
+    def add(self,item_to_added):
+        self.inventory.append(item_to_added)
+        return item_to_added
+    
+    def remove(self,item_to_removed):
+        if item_to_removed not in self.inventory:
+            return False
+
+        self.inventory.remove(item_to_removed)
+        return item_to_removed
+
+    def get_by_id(self, id):
+        #loop through the item in inventory
+        #if id input is equal to the item id in inventory then return item
+        for item in self.inventory:
+            if id == item.id:
+                return item
+        return None
+    
+    def swap_items(self, other_vendor, my_item, their_item):
+        if my_item not in self.inventory or their_item not in other_vendor.inventory:
+            return False
+        
+        self.remove(my_item)
+        other_vendor.add(my_item)
+
+        other_vendor.remove(their_item)
+        self.add(their_item)
+        return True
+
+    def swap_first_item(self, other_vendor):
+        #check to see we have empty inventory
+        #if so then return False
+        if len(self.inventory) == 0 or \
+        len(other_vendor.inventory) == 0:
+            return False
+        else:
+            #grab the first item in our inventory and assign to variables
+            my_swap = self.inventory[0]
+            friend_swap = other_vendor.inventory[0]
+            
+            # Replace the first item in our inventory with the first item from the other inventory  
+            self.inventory[0] = friend_swap
+            other_vendor.inventory[0] = my_swap
```

Comment:
You have rightfully recognized that our tests aren't checking to see where the items get swapped to. We could use our `swap_items()` method here, or we could do what y'all did and simply swap the first items of each list by position! I think y'all's approach is a smart move, but we do have one trick we could use to take the swap to the next level: multiple assignment. We can assign multiple values at once! 

```suggestion
            self.inventory[0], other_vendor.inventory[0] = \
                other_vendor.inventory[0], self.inventory[0]
```